src/jit/gentree.cpp

```diff
@@ -1740,6 +1740,35 @@ bool GenTreeCall::IsHelperCall(Compiler* compiler, unsigned helper) const
     return IsHelperCall(compiler->eeFindHelper(helper));
 }
 
+void GenTreeCall::ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement)
+{
+    assert(operandUseEdge != nullptr);
+    assert(replacement != nullptr);
+    assert(TryGetUse(*operandUseEdge, &operandUseEdge));
+
+    GenTree* originalOperand = *operandUseEdge;
+    *operandUseEdge = replacement;
+
+    const bool isArgument = (replacement != gtControlExpr) && ((gtCallType != CT_INDIRECT) ||
+        ((replacement != gtCallCookie) && (replacement != gtCallAddr)));
```

Comment:
I think it's easy to get confused between "argument" which is here referring only to the actual call argument nodes, and "operand" which is what these other things are. It wouldn't hurt to add a comment like "Is this an operand that is contained in an argument list?"

---
In reply to: [78810372](https://github.com/dotnet/coreclr/pull/7170#discussion_r78810372) [](ancestors = 78810372)